src/System.Private.ServiceModel/tools/test/Bridge/WcfTestBridgeCommon/CertificateGenerator.cs

```diff
@@ -35,7 +35,8 @@ public class CertificateGenerator
 
         // This can't be too short as there might be a time skew between machines,
         // but also can't be too long, as the CRL is cached by the machine
-        private TimeSpan _crlValidityPeriod = TimeSpan.FromMinutes(2);
+        private TimeSpan _crlValidityGracePeriodStart = TimeSpan.FromMinutes(5);
+        private TimeSpan _crlValidityGracePeriodEnd = TimeSpan.FromMinutes(5);
 
         // Give the cert a grace period in case there's a time skew between machines
         private readonly TimeSpan _gracePeriod = TimeSpan.FromHours(1);
```

Comment:
If there is a time skew between machines, should we be using the "now" of the client (i.e. pass it in as part of the request)?  It might be worth considering (not in this PR) how to compute start and end times that are guaranteed to be consistent with whichever machine is going to validate them (or that has a start time of the earliest of the 2 times and an end time of the latest) to guarantee validity on both machines.